scripts/install

```diff
@@ -52,33 +57,30 @@ chown $app:$app "$install_dir/custom/conf/app.ini"
 ynh_script_progression --message="Upgrading systemd configuration..." --weight=1
 
 ynh_add_systemd_config
-
-#=================================================
-# NGINX CONFIGURATION
-#=================================================
-ynh_script_progression --message="Upgrading NGINX web server configuration..." --weight=2
+ynh_add_systemd_config --service=${app}_runner --template="runner.service"
 
 ynh_add_nginx_config
 
-#=================================================
-# GENERIC FINALIZATION
-#=================================================
-# SETUP LOGROTATE
-#=================================================
-ynh_script_progression --message="Configuring log rotation..." --weight=1
-
 # Use logrotate to manage application logfile(s)
 ynh_use_logrotate --logfile "/var/log/$app" --nonappend
 
 chown -R $app:$app "/var/log/$app"
 chmod u=rwX,g=rX,o= "/var/log/$app"
 
+yunohost service add $app --description="Forgejo" --log="/var/log/$app/forgejo.log"
+yunohost service add ${app}_runner --description="Forgejo runner" --log="/var/log/$app/forgejo.log"
+
 #=================================================
-# INTEGRATE SERVICE IN YUNOHOST
+# ADD A CONFIGURATION
 #=================================================
-ynh_script_progression --message="Integrating service in YunoHost..." --weight=2
+ynh_script_progression --message="Adding a configuration file..." --weight=1
 
-yunohost service add $app --description="Forgejo" --log="/var/log/$app/forgejo.log"
+secret=$(openssl rand -hex 20)
```

Comment:
Using `openssl` produces a warning during CI.
`ynh_string_random` should be preferred. Or better : `forgejo actions generate-runner-token`.

This secret should be probably saved in the app settings.